types/luaparse/lib/ast.d.ts

```diff
@@ -0,0 +1,297 @@
+export const enum Type {
```

Comment:
Do not use const enum here, There doesn't exist const of name 'Type' in your library.

Reply:
Resolved - const enum is removed.